multi-agent/lib/mas/blueprints/service.py

```
@@ -159,10 +175,40 @@ def get_draft_schema() -> Dict[str, Any]:
         """
         return BlueprintDraft.model_json_schema()
 
+    # ────────── Prompt Shortcuts ──────────
+    def set_prompt_shortcuts(self, blueprint_id: str, prompts: list[dict],
+                             *, identity: Identity) -> PromptShortcuts:
+        try:
+            doc = self._repo.load(blueprint_id)
+        except KeyError:
+            raise BlueprintNotFoundError(blueprint_id)
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)